lib/src/easy_smart_banner/easy_smart_banner.dart

```diff
@@ -0,0 +1,72 @@
+import 'dart:async';
+
+import 'package:easy_ads_flutter/easy_ads_flutter.dart';
+import 'package:flutter/material.dart';
+
+class EasySmartBannerAd extends StatefulWidget {
+  const EasySmartBannerAd({Key? key}) : super(key: key);
+
+  @override
+  State<EasySmartBannerAd> createState() => _EasySmartBannerAdState();
+}
+
+class _EasySmartBannerAdState extends State<EasySmartBannerAd> {
+  final adNetworks = [
+    AdNetwork.admob,
+    AdNetwork.facebook,
+    AdNetwork.appLovin,
+    AdNetwork.unity,
+  ];
```

Comment:
this will be input from user. PriorityAdNetorks and provide a default list as well